docs/Installing.rst

```diff
@@ -91,19 +63,22 @@ restrictions apply (e.g. a file named ``Windows64`` is for 64-bit Windows DF).
 Installing DFHack
 =================
 
+If you are installing from Steam, this is handled for you automatically. The
+instructions here are for manual installs.
+
 When you `download DFHack <downloading>`, you will end up with a release archive
 (a ``.zip`` file on Windows, or a ``.tar.bz2`` file on other platforms). Your
 operating system should have built-in utilities capable of extracting files from
 these archives.
 
-The release archives contain several folders, including a ``hack`` folder where
-DFHack binary and system data is stored, a ``dfhack-config`` folder where user
-data and configuration is stored, and a ``blueprints`` folder where `quickfort`
-blueprints are stored. To install DFHack, copy all of the files from the DFHack
-archive into the root DF folder, which should already include a ``data`` folder
-and a ``raw`` folder, among other things. Some packs and other redistributions
-of Dwarf Fortress may place DF in another folder, so ensure that the ``hack``
-folder ends up next to the ``data`` folder.
+The release archives contain a ``hack`` folder where DFHack binary and system
+data is stored, a ``stonesense`` folder that contains data specific to the
+`stonesense` 3d renderer, and various libraries and executable files. To
+install DFHack, copy all of the files from the DFHack archive into the root DF
+folder, which should already include a ``data`` folder and a ``mods`` folder,
```

Comment:
the `mods` folder may not be present if the player has never used mods - might be best to state that this folder _may_ be present

Reply:
I could mention `save` instead. That might be more universal